src/app2/app2.js

```diff
@@ -11,6 +12,7 @@ const ngLifecycles = singleSpaAngular({
   angularPlatform: platformBrowserDynamic(),
   template: `<app2 />`,
   Router,
+  ApplicationRef,
```

Comment:
This is new as of single-spa-angular@2.1.0. Providing it is necessary for the bug fix to happen.